README.md

```diff
@@ -19,6 +20,14 @@ Go to [excalidraw.com](https://excalidraw.com) to start sketching.
 
 Read the latest news and updates on our [blog](https://blog.excalidraw.com). A good start is to see all the updates of [One Year of Excalidraw](https://blog.excalidraw.com/one-year-of-excalidraw/).
 
+## We accept donations
+
+If you like the project, you can become a sponsor at [Open Collective](https://opencollective.com/excalidraw).
```

Comment:
```suggestion
If you like the project, you can contribute to our costs of running the infrastructure by becoming a sponsor at [Open Collective](https://opencollective.com/excalidraw).
```

maybe, so that new users who may not be aware that Excalidraw has a backend that costs money?

Reply:
mixed feeling for that.. dunno who doesn't know that things needs electricity to keep it running :)

Reply:
I liked the first shorter version better.

Reply:
I meant, it's not immediately obvious we have WS and Storage servers. But let's leave it then.